element-hq/element-x-android
# Mark room as unread

## Type of change


## Content


Add action to mark room as read / un-read from the room list.

## Motivation and context


Closes #2261 

## Screenshots / GIFs


See recorded screenshots.

## Tests



- Make sure that the feature "Mark as unread" is enabled in the developer settings
- From the room list, long click on a room
- Select mark as read / mark as unread.
- When marked as unread, clicking on the room to see the timeline will mark the room as read (and eventually send the read receipt.

Private or public read receipt will be used according to current user setting.

- If the feature "Mark as unread" is disabled in the developer settings, the option to mark as read / mark as unread are not displayed.

## Tested devices

- OS version(s):

## Checklist
